help/en/Acknowledgements.shtml

```diff
@@ -631,6 +631,10 @@
 
         <li>John McAleely, who contributed Hornby and Bachmann decoder definitions</li>
 
+        <li>Jordan McBride, <!-- TCSDCC-Jordan -->
+        who worked on the TCS decoder identification, new split value variable form,
+        and TCS decoder definitions.
```

Comment:
I don't know if it matters, but Jordan McBride has deleted his GitHub account. See https://github.com/JMRI/JMRI/pull/10568#issuecomment-1011216075.
So `TCSDCC-Jordan` is not a valid GitHub user anymore.

Reply:
Thanks.  I think I'll leave that in place, as there are still some references to the account around.